shared/icons/icons.ts

```diff
@@ -1,4 +1,5 @@
 export const IconName = {
+  CHECK_MARK_SOLID: "check-mark-solid" as "check-mark-solid",
```

Comment:
No comment or request change here.
Just want to say that the code is really coming together nicely, following a logical and readable flow.
Good job!!! :-) 🥇